packages/components/src/spinners/CenteredSpinner.tsx

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash/fp';
+import React, { CSSProperties, ReactNode } from 'react';
+
+import { Spinner, SpinnerProps } from './Spinner';
+
+export const CenteredSpinner = (props: SpinnerProps = {}): ReactNode => {
```

Comment:
Now that I think about it though, I think the default value for `props` is a holdover from the render function. Looking at some other components, none of them have default values for `props` itself.  The props object is always passed by React.
https://react.dev/learn/passing-props-to-a-component

Getting rid of the default value here would resolve the coverage issue.

```suggestion
export const CenteredSpinner = (props: SpinnerProps): ReactNode => {
```

Reply:
ok, I like this outcome alot. thx!